src/Files.App/UserControls/InnerNavigationToolbar.xaml.cs

```diff
@@ -1,64 +1,72 @@
 // Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using Files.App.Data.Commands;
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Input;
 using Microsoft.UI.Xaml.Media.Imaging;
 using System.IO;
 
-// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
-
 namespace Files.App.UserControls
 {
 	public sealed partial class InnerNavigationToolbar : UserControl
 	{
-		public InnerNavigationToolbar()
-		{
-			InitializeComponent();
-			PreviewPaneViewModel = Ioc.Default.GetRequiredService<PreviewPaneViewModel>();
-		}
+		private readonly IAddItemService _addItemService = Ioc.Default.GetRequiredService<IAddItemService>();
```

Comment:
How come?

[Runtime Code Style Conventions](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) Itme 3:

> We use _camelCase for internal and private fields and use readonly where possible.

[Our Code Stlye Guideline](https://files.community/docs/contributing/code-style) Section 2, Item 4:

> Use camelCase prefixed with _ for private fields

Reply:
In that case, the two services below are not named properly.

Reply:
Yeah, I agree with you. I will fix.

But as my policy ones that are referenced from XAML should be named with PascalCase; Commands & ModifiableCommands because when accessing in XAML, the code will be like as follows if CommandManager is defines in code-behind.
```xaml
<Button Command="Commands.OpenInNewTab" />
```

Is that not bad practice? I wanna hear from you.

Reply:
There is something off with the second format indeed, it looks really off. Yet at the same time, if we're making an exception to our coding rules, than I believe the coding rules are not good enough.

On a first read, one can be confused as to why some private fields are marked with an underscore and why others are not. 

Maybe we can mark the fields used in xmal with a letter (bad idea in my opinion). Maybe we can add a rule in the coding rules about having them capitalized as a way to mark their use in XMAL (will work for a while, but give it a year or two and the whole thing gonna be a real mess).

I need to think about it a little bit to see if anything clever can be made. Maybe we can find some inputs online for good practice.

Either way, that is something minor here. We can open an issue about that to keep track of it.

Tagging @d2dyno1 since I believe he wrote the coding rules, he might have some good input about it?

Reply:
Making them properties would fix issues. How about?

Reply:
Why not just use a private property in this case?